src/core/microservices/subscription.factory.provider.ts

```diff
@@ -5,12 +5,28 @@ import { ConfigService } from '@nestjs/config';
 import { subscriptionFactory } from '@core/microservices/subscription.factory';
 import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
 
+/***
+ * Creates a factory provider for a subscription
+ * @param provide The injector token
+ * @param queueName Queue name to which to subscribe
+ * @param exchangeName Name of the exchange
+ * @param trackingID Suffix to concatenate to the queue name and exchange.
+ * This can be used for creating different queues and exchanges for different replicas of the running service.
+ * Pro tip: Assigning a suffix per replica will improve tracking of queues. This can be further improved by passing the k8s replica UUID.
```

Comment:
```This can be further improved by passing the k8s replica UUID``` - what do you mean by that?

Reply:
It will be better to know which replica created this queue for tracking purposes - it's a pro tip.